### ISSUE TITLE
Pressing mouse in the search box, then releasing it over the sticky scroll, unfocuses the search box

### ISSUE THREAD

**Issue by HolyBlackCat** [microsoft/vscode#181200]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.77.3
- OS Version: Xubuntu 22.04

Steps to Reproduce:

1. Trigger sticky scroll.
2. Ctrl+F, then type something in the search box.
3. Try to select the searched text with mouse, liks this:
   * Click and hold in the textbox, to the right of the text.
   * Drag the mouse to the left.
   * Overshoot the search box, and release the mouse to the left of it, on top of the sticky scroll.
4. Observe that the textbox is unfocused, and the sticky scroll navigation is triggered.

Instead, the textbox should remain focused, and sticky scroll navigation shouldn't happen.

In other words, sticky scroll should only handle mouse release IF the mouse was previously pressed on top of it.

**Comment by Clarkkkk** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/174205
